cpp/ycm/ClangCompleter/Diagnostic.h

```diff
@@ -53,6 +81,8 @@ struct Diagnostic {
   std::string text_;
 
   std::string long_formatted_text_;
+
+  std::vector< FixItChunk > fixits_;
```

Comment:
I'm probably missing something, but why is this not a vector of `FixIt`s instead of `FixItChunk`s?